tests/components/Loader.test.js

```diff
@@ -0,0 +1,31 @@
+import Loader from '../../src/components/Loader/Loader'
```

Comment:
the import statement, and please reverse import order.

Reply:
done